subprojects/packagefiles/jbigkit/libjbig/meson.build

```diff
@@ -0,0 +1,16 @@
+sources = files('jbig.c', 'jbig_ar.c')
+sources85 = files('jbig85.c', 'jbig_ar.c')
+
+libjbig = library('jbig', sources)
+libjbig_dep = declare_dependency(
+	include_directories: include_directories('.'),
+	link_with: libjbig,
+	install: true)
```

Comment:
```
jbigkit| subprojects/jbigkit-2.1/libjbig/meson.build:5: WARNING: Passed invalid keyword argument "install".
jbigkit| WARNING: This will become a hard error in the future.
```

This kwarg would need to be added to library(), not declare_dependency